README.rst

```diff
@@ -183,6 +163,65 @@ Let us check the results:
 
     $ firefox worldpopulation/world_population_analysis.html
 
+Local testing with CWL
+=========================
+
+Let us test whether the CWL workflow execution works locally as well.
```

Comment:
The example works nicely for me :+1: 

One documentation comment to simplify this part; please see https://github.com/reanahub/reana-demo-helloworld/pull/17/files#r170864827